package.json

```diff
@@ -69,6 +69,7 @@
     "js-yaml": "3.13.1",
     "proxy-agent": "5.0.0",
     "rimraf": "^3.0.2",
+    "semver": "^7.3.7",
```

Comment:
Could you run `yarn` and push the changes to the lockfile `yarn.lock`?

Reply:
👍 Added and pushed changes.